tools/run-clang-tidy.sh

```diff
@@ -20,7 +20,11 @@
 
 set -o errexit -o xtrace
 
-ROS_PACKAGES="beluga beluga_system_tests"
+if [ "${ROS_DISTRO}" != "noetic" ]; then
+    ROS_PACKAGES="beluga beluga_system_tests"
+else
+    ROS_PACKAGES="beluga"
```

Comment:
Regarding the CI static analysis failure, we can probably add `-clang-diagnostic-gnu-zero-variadic-macro-arguments` to the checks in `.clang-tidy`.

The include order warning is weird, but adding a newline in between in `test_strongly_typed_numeric.cpp` may fix it:
```cpp
#include <gtest/gtest.h>

#include <beluga/type_traits/strongly_typed_numeric.hpp>
```

Reply:
Hmm, there is also an unused parameter in a googletest macro... I wonder if it is related to the googletest version or the differences between `clang-tidy-10` and `clang-tidy-14`.

Reply:
I believe it's https://github.com/google/googletest/issues/2650, which has been fixed for googletest 1.11 but not backported to 1.10. I tried and failed to handle googletest versions in CMake -- Kitware's GTest module doesn't detect any versions. Dropping the warning seems like the only path forward.

Reply:
>  adding a newline in-between in test_strongly_typed_numeric.cpp may fix it:

Indeed.

Reply:
> we can probably add -clang-diagnostic-gnu-zero-variadic-macro-arguments to the checks in .clang-tidy.

This doesn't seem to work. Passing `-Wno-gnu-zero-variadic-macro-arguments` to the compiler does (and thus to the compiler_commands.json file) does.